Corona-Warn-App/src/main/java/de/rki/coronawarnapp/submission/DaysSinceOnsetOfSymptomsVectorDeterminator.kt

```diff
@@ -0,0 +1,54 @@
+package de.rki.coronawarnapp.submission
+
+import dagger.Reusable
+import de.rki.coronawarnapp.util.TimeAndDateExtensions.ageInDays
+import de.rki.coronawarnapp.util.TimeAndDateExtensions.toLocalDate
+import de.rki.coronawarnapp.util.TimeStamper
+import javax.inject.Inject
+
+@Reusable
+class DaysSinceOnsetOfSymptomsVectorDeterminator @Inject constructor(
+    private val timeStamper: TimeStamper
+) {
+
+    @Suppress("MagicNumber")
+    internal fun determine(symptoms: Symptoms): DaysSinceOnsetOfSymptomsVector {
+        return when (symptoms.symptomIndication) {
+            Symptoms.Indication.POSITIVE ->
+                determinePositiveIndication(symptoms)
+            Symptoms.Indication.NO_INFORMATION ->
+                createDaysSinceOnsetOfSymptomsVectorWith(4000)
```

Comment:
Would an enum or companion object be over-engineered. Currently thinking about all the "Magic numbers" to move to an enum or companion object. @chris-cwa, @d4rken, @ralfgehrer would you guys think?

Reply:
IMHO that's over-engineered. They are rather magical ;)

Reply:
Not sure an enum is necessary (leave it up to you) but if not, a class-level (Javadoc) comment might be appropriate here to give a rough explanation what these magic numbers mean and where they come from. Let me know if you need any support with that.

Reply:
I will add a link to your documentation, @mlenkeit